docs/UserGuide.adoc

```diff
@@ -158,68 +173,70 @@ You can find a contact added to the address book easily with a name.
 
 Format: `find n\[name]`
 
-[NOTE]
-`[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-
-Example: +
-`find n\john DOE`
-
-==== Add Picture to Contact: `addpic`
-You can add a profile picture to a contact in your address book.
-
-Format: `addpic n\[name] pa\[path]`
-
 [NOTE]
 ====
 * `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-* Absolute file path is used for `[path]`.
+* You can find partial name
 ====
 
-Example: +
-`addpic n\john doe pa\C:\Users\aloha\Desktop\aloha.png`
+Examples:
 
-Sets the profile picture of John Doe to aloha.png.
+* `find n\john DOE` +
+Finds a saved contact named John Doe.
 
-==== Update Picture to Contact: `updatepic`
-You can update the profile picture of a contact in your address book.
+* `find n\john` +
+Finds a saved contact named John.
+
+==== Add Picture to Contact: `addpic`
+You can add a profile picture to a contact in your address book.
 
-Format: `updatepic n\[name] pa\[path]`
+Format: `addpic INDEX pa\[path]`
 
 [NOTE]
 ====
-* `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+* Adds profile picture of the person at the specified `INDEX`. The index refers to the index number shown in
+the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * Absolute file path is used for `[path]`.
 ====
 
 Example: +
-`updatepic n\john doe pa\C:\Users\aloha\Desktop\updatedAloha.png`
+`addpic 2 pa\C:\Users\aloha\Desktop\aloha.png`
 
-Change the picture of John Doe to updateAloha.png.
+Sets the profile picture of contact at index 1 to aloha.png.
 
 ==== Delete Picture in Contact: `deletepic`
 You can delete the profile picture of a contact in your address book.
 
-Format: `deletepic n\[name]`
+Format: `deletepic INDEX`
 
 [NOTE]
-`[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+====
+* Deletes the picture at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+====
 
 Example: +
-`deletepic n\john doe`
-Deletes the profile picture of John Doe
+`deletepic 2`
+
+Deletes the profile picture contact at index 1.
```

Comment:
You deleted the picture of 2, then suddenly becomes index 1 O.o? Also, use layman terms. "Deletes the profile picture of the first/second contact in your address book.